lib/posix-netlink/netlink.c

```diff
@@ -277,6 +277,9 @@ static ssize_t nl_recvfrom(posix_sock *s,
 	struct nl_ctx *nl_ctx = sock2nlctx(s);
 	size_t cpylen;
 
+	if (from && !fromlen)
```

Comment:
This commit made me check and actually should we also check for `!buf` and return `-EFAULT`?